### ISSUE TITLE
isChatOnline & lastSeen Error

### ISSUE THREAD

**Issue by ozoktay** [open-wa/wa-automate-nodejs#324]
İf i can use getChatById always return **isOnline: null,
  lastSeen: null**.

`async function start(client) {
	var xxx = await client.getChatById('xxxxxxxxx@c.us');
	console.log(xxx);
}`

İf i can use isChatOnline always return **false**.

`async function start(client) {
	var xxx = await client.getChatById('xxxxxxxxx@c.us');
	console.log(xxx);
}`

I'm using the latest version.

**Comment by ozoktay** [comment]
https://github.com/open-wa/wa-automate-nodejs/issues/279 is still working only if has conversation.

How can I check this for any number in my contacts?

**Comment by smashah** [comment]
Thanks for bringing this up. Will be fixed in a bit.

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i`);
exec(`npm run release-ci`);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);

var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [1.8.60](https://github.com/open-wa/wa-automate-nodejs/compare/1.8.59...1.8.60) (2020-04-15)

- Release 1.8.60 [`78d158e`](https://github.com/open-wa/wa-automate-nodejs/commit/78d158eb1769cef805aa26b27a4c169886b1e1eb)
- implemented `getLastSeen` #324 [`4405e07`](https://github.com/open-wa/wa-automate-nodejs/commit/4405e07abc19863b784ce96c1464a25d4b24f246)
- updated docs with release image [`3cdfafd`](https://github.com/open-wa/wa-automate-nodejs/commit/3cdfafd3120af88d742edfcc6eb70d3547fe713d)
- updated release-image [`2674aa6`](https://github.com/open-wa/wa-automate-nodejs/commit/2674aa620e40d63f0f2fe90564aa1545a57e04f8)

**Comment by smashah** [comment]
@ozoktay lastSeen and isOnline on the chat object are now deprecated. Please use:

```javascript
const lastSeen = await client.getLastSeen('4477777777777@c.us');
const isOnline = await client. isChatOnline('4477777777777@c.us');
```

You cannot get the last seen or online statuses of contacts you don't already have chats with. Also please see all the caveats in the docs:

https://open-wa.github.io/wa-automate-nodejs/classes/whatsapp.html#getlastseen

This has been tested and is working.
